community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/TransactionStateMachine.java

```diff
@@ -47,6 +49,7 @@
     private static final Pattern BEGIN = Pattern.compile("(?i)^\\s*BEGIN\\s*;?\\s*$");
     private static final Pattern COMMIT = Pattern.compile("(?i)^\\s*COMMIT\\s*;?\\s*$");
     private static final Pattern ROLLBACK = Pattern.compile("(?i)^\\s*ROLLBACK\\s*;?\\s*$");
+    private static final String TX_TIMEOUT_MILLIS = "txTimeout";
```

Comment:
Maybe "timeout" instead of "txTimeout"? We already have "bookmark" and "bookmarks" without "tx" prefix.